### ISSUE TITLE
[Idea] Update wiki page on the GitHub repo

### ISSUE THREAD

**Issue by Splarkszter** [ORelio/Minecraft-Console-Client#2377]
### Prerequisites

- [X] I have read and understood the [user manual](https://mccteam.github.io/guide/)
- [X] I made sure I am running the latest [development build](https://github.com/MCCTeam/Minecraft-Console-Client/releases/latest)
- [X] I tried to [look for similar feature requests](https://github.com/MCCTeam/Minecraft-Console-Client/issues?q=is%3Aissue) before opening a new one

### Console Client Version

n/a

### Describe your problem

there is no way to find "https://mccteam.github.io/" unless you sign-in to discord, enter the MCC server and go to the '#info' channel, instead you get this. Which can confuse people or make them think there is no documentation at all (Well, that will only happen if you never scroll down on the Home/Code tab of this repo and instead you go directly to the Wiki tab[i did forgot about that and did that. lmao]).

### Suggest a possible solution

Update the Wiki tab so there is at least a link to the Docs page.

### Attach screenshot here (If applicable)

![image](https://user-images.githubusercontent.com/90066319/206918599-0ab75557-0ee0-4626-83a4-9bf209d3995b.png)

### Minecraft Version

n/a

### Device

None

### Operating System

None

**Comment by Splarkszter** [comment]
It's not necessary but it's a tiny a Quality of Life change

**Comment by milutinke** [comment]
It's linked in the README.md on multiple places, I do not know how you've missed it.

**Comment by milutinke** [comment]
Feel free to open a PR and tweak the wiki with your idea.

**Comment by breadbyte** [comment]
> Feel free to open a PR and tweak the wiki with your idea.

I don't think you can make pull requests against the github wiki page, only contributors can modify the page.

**Comment by milutinke** [comment]
> > Feel free to open a PR and tweak the wiki with your idea.
> 
> I don't think you can make pull requests against the github wiki page, only contributors can modify the page.

Oh, ok, I'll add it when I catch some free time.
